pkgs/desktops/kde-4.14/kdebindings/pykde4.nix

```diff
@@ -0,0 +1,35 @@
+{ kde, kdelibs, pythonPackages, kdepimlibs, shared_desktop_ontologies,
```

Comment:
Python bindings are just Python libraries and so should therefore be called from `python-packages.nix`, with the expression also living in `python-modules`.

Reply:
@FRidh Done, thank you!